### PR TITLE
Use static URL method and create respons based on request status

### PR DESCRIPTION
**Pull request by elekjani**

### CODE REVIEW

File: son-gtkapi/models/vim_manager_service.rb [hunk 1]
```diff
@@ -42,53 +42,58 @@ def self.config(url:, logger:)
...
     method = LOG_MESSAGE + "##{__method__}(#{params})"
     @@logger.debug(method) {"entered"}
-    
+
     begin
-      @@logger.debug(method) {"@url = "+@url}
-      #response = RestClient.post(@url+'/vim', params.to_json, content_type: :json, accept: :json) 
-      response = postCurb(url: @@url+'/vim', body: params.to_json) 
+      @@logger.debug(method) {"@url = " + self.url}
+      response = postCurb(url:self..url+'/vim', body: params.to_json)
```
Comment (jbonnet) [inline son-gtkapi/models/vim_manager_service.rb]: `self..url`will not work

File: son-gtkapi/models/vim_manager_service.rb [hunk 2]
```diff
@@ -42,53 +42,58 @@ def self.config(url:, logger:)
...
   end
-  
+
   def self.find_vim_request_by_uuid(uuid)
     method = LOG_MESSAGE + "##{__method__}(#{uuid})"
     @@logger.debug(method) {'entered'}
     begin
-      response = getCurb(url:@@url+'/vim_request/'+uuid, headers: JSON_HEADERS) 
-      JSON.parse response.body
+      response = getCurb(url:self.url+'/vim_request/'+uuid, headers: JSON_HEADERS)
```
Comment (jbonnet) [inline son-gtkapi/models/vim_manager_service.rb]: Please use plurals, like in `vim_requests`

Comment (jbonnet) [inline son-gtkapi/models/vim_manager_service.rb]: (I know this was already implemented as singular, but since you're touching it...) ;-)

**Comment by elekjani** [comment]
Hi @jbonnet, thanks for the review, I've created a new commit accordingly.

**Comment by elekjani** [comment]
Hi @jbonnet, I've found some additional bug in the code, and introduced the 202 status code instead of 206 as I think it describes better the situation with the requests. What do you think? Also, should I commit in a different branch and create different pull requests for different commits or is it OK that way? As I see these commits are related so it can be put in one pull request, what is your opinion on that?

**Comment by jbonnet** [comment]
Hi, @elekjani `202`as a `status code` should be fine (we need to improve that all over the code base, mostly only `200` is being used). It's ok to commit here, don't worry. I'll take a look.

Many thanks

**Pull request merged**